apps/frontend/src/components/global/Footer.vue

```diff
@@ -3,6 +3,18 @@
     <span class="bar-visible--text">
       The MITRE Corporation &copy; 2018-{{ new Date().getFullYear() }}
     </span>
+    <v-spacer />
+    <a href="https://www.netlify.com">
+      <img
+        :src="require('@/assets/netlify-light.svg')"
+        style="width: 1.5rem; height: 1.5rem; vertical-align: middle"
+        class="mr-4"
+        alt="Netlify.com"
+      />
+    </a>
+    <a href="https://github.com/mitre/heimdall2" style="text-decoration: none">
+      <v-icon medium color="white">mdi-github</v-icon>
+    </a>
```

Comment:
while we're making changes here, can you add a button for releases+version number similar to the load modal.  put that button on the far right.

desired order:
left aligned: copy right notice
right aligned: github, netlify, release+version